CoreScriptsRoot/Modules/DeveloperConsoleModule.lua

```diff
@@ -2990,15 +2990,24 @@ local function onDevConsoleVisibilityChanged(isVisible)
 	end
 end
 
+local getDeveloperConsoleIsCreating = false
 local function getDeveloperConsole()
-	if not myDeveloperConsole then
-		local permissions = DeveloperConsole.GetPermissions()
-		local messagesAndStats = DeveloperConsole.GetMessagesAndStats(permissions)
+	if (not myDeveloperConsole) then
+		if (not getDeveloperConsoleIsCreating) then
```

Comment:
this is fine, although it seems weird that new must yield at some point... we should look into this in the future.